src/logging/config/LogConfiguration.cpp

```diff
@@ -18,12 +18,13 @@ LogConfiguration::LogConfiguration(
   tagLog.setDocumentation("Configures logging sinks based on Boost log.");
 
   auto attrLogEnabled = makeXMLAttribute("enabled", true)
-                            .setDocumentation("Enables logging");
+                            .setDocumentation("Enables the creation of log sinks. Disable sinks if you plan to setup Boost.log in your application.");
```

Comment:
I'm not sure I get this sentence right: with the proposed solution, you could still use Boost.log in your own application without conflicting sinks. Why should I now disable the sinks now?

Reply:
If an application uses boost.log then it has two choices:

* Use the preCICE provided sinks (default) and remove the preCICE log entries from the application-sinks to provide duplication.
* Or it can handle both the application and the preCICE logs, then it would need to disable the sinks of preCICE, which is what this attribute does.

Feel free to help with a better formulation. I find it confusing :sweat_smile:

Reply:
Maybe then `Disable sinks if you want to handle the preCICE logging in your application`? I guess a link to a dev doc or to boost log might be helpful.

Reply:
I updated the text. I would really avoid links from the xmlrefence to the precice website. All of this behaviour should be in the real docs.